docs/docs/aleph_zero/sdk.md

```diff
@@ -10,31 +10,55 @@ SDK (Software Development Kit) can be used to interact with our DEX programatica
+3. Wrapped Native Currency Contract (Wrapped AZERO): This contract allows users to wrap the native currency, providing compatibility and interoperability within the Invariant ecosystem.
 
-### Event listeners
+### Transactions and Queries
 
-Invariant class has additional methods that allows you to attach event listeners so you can run your own code based on contract activity.
+When working with contracts, developers can seamlessly initiate interactions by calling methods from the corresponding contract class. The first parameter designates the account, and subsequent parameters act as entrypoint parameters, facilitating smooth contract interactions.
+
+1. **Transactions**: These involve invoking methods that result in changes to the blockchain state. Transactions typically alter the data stored on the blockchain and may include operations like transferring assets, updating records, or executing specific actions.
+
+2. **Queries**: Unlike transactions, queries are read-only interactions with the contract. They allow developers to retrieve information from the blockchain without modifying its state. Queries are useful for obtaining current contract state, fetching data, or verifying certain conditions without making any changes to the blockchain.
+
+### Listening to Events
+
+The Invariant class offers additional methods empowering developers to attach event listeners. This feature enables the execution of custom code based on contract activity, enhancing flexibility and customization in response to specific events.
 
 ### Values and Helper functions
 
-SDK also contains all essential values and helper functions that might be needed in your app like maximum tick, maximum price, calculating price impact and many others that will simplify your calculations.
+The SDK includes fundamental values and utility functions for application development. These encompass parameters such as maximum tick, maximum price, and calculations for price impact. This suite of utility functions streamlines computational aspects, enhancing the efficiency of application logic.
 
-## Files
+### Contracts Metadata
+
+Within the Contracts folder, developers can find deploy-ready contracts and metadata essential for interactions. This structure facilitates straightforward integration with the specified contracts, simplifying the development workflow.
+
+### Math
+
+The Math folder contains Rust structs and data exported to WebAssembly using wasm_bindgen. This integration allows for the utilization of the same functions and data employed in the primary Invariant contract.
+
+### Source
+
+The Source directory consolidates all pieces into an easy-to-use interface. This organized structure simplifies the development process and provides developers with a centralized location for accessing essential resources.
+
+### Tests
+
+The Tests folder houses test suites to ensure the correct functioning of SDK.
```

Comment:
Here's a general note for the entire SDK docs. It's better to provide more details that contribute technical information rather than general definitions that apply to all projects.

For example: "The Tests folder houses test suites to **ensure the correct functioning of the SDK**."
This is the purpose of all tests and applies to every project, so it doesn't provide specifications about this project. Specific details include:
- These are e2e tests
- Framework used by the tests
- Which sdk functions was tested
- ...